### DISCUSSION TITLE
这个没有独显可以运行吗

### DISCUSSION THREAD

**Discussion by cgrjfk** [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN#43] [General]
这个没有独显可以运行吗

**Comment by PengfeiM** [comment]
需要N卡的

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Lucy.J.Smith ***@***.***>
发送时间: Saturday, March 11, 2023 9:32:21 PM
收件人: PengfeiM/Fatigue-Driven-Detection-Based-on-CNN ***@***.***>
抄送: Subscribed ***@***.***>
主题: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)


这个没有独显可以运行吗

―

**Reply by PUPPY2021** [reply]
不用N卡在云服务器上能运行吧

**Comment by cgrjfk** [comment]
好吧，谢谢啦发自我的荣耀平板-------- 原始邮件 --------发件人： Ma Pengfei ***@***.***>日期： 2023年3月14日周二 晚上9:48收件人： PengfeiM/Fatigue-Driven-Detection-Based-on-CNN ***@***.***>抄送： "Lucy.J.Smith" ***@***.***>, Author ***@***.***>主    题： Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)
需要N卡的



从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送



________________________________

发件人: Lucy.J.Smith ***@***.***>

发送时间: Saturday, March 11, 2023 9:32:21 PM

收件人: PengfeiM/Fatigue-Driven-Detection-Based-on-CNN ***@***.***>

抄送: Subscribed ***@***.***>

主题: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)





这个没有独显可以运行吗



―

**Comment by cgrjfk** [comment]
是的 不用n卡其实也没啥问题 只要升级pytorch版本 英特尔的显卡也能训练，不过这个问题我已经好几年前问过了，现在受到回复还是蛮有趣的😂😂












东耳国王
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年3月11日(周二) 下午5:18
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)




 
不用N卡在云服务器上能运行吧

**Comment by PUPPY2021** [comment]
我指的是用amd核显或者用云端服务器运行




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PengfeiM/Fatigue-Driven-Detection-Based-on-CNN"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年3月11日(星期二) 晚上10:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)





 是的 不用n卡其实也没啥问题 只要升级pytorch版本 英特尔的显卡也能训练，不过这个问题我已经好几年前问过了，现在受到回复还是蛮有趣的😂😂 
 
 
 
 
 
 
 
 
 
 
 
 
 东耳国王 
 ***@***.*** 
 
 
 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2025年3月11日(周二) 下午5:18 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43) 
 
 
 
 
   
 不用N卡在云服务器上能运行吧

**Comment by cgrjfk** [comment]
这个amd核显运行起来非常慢 至于云服务器 配置好了就行了 其实其他显卡一样可以训练，很多开源库可以实现 只是比较麻烦











---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年3月13日(周四) 下午2:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)




 我指的是用amd核显或者用云端服务器运行 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "PengfeiM/Fatigue-Driven-Detection-Based-on-CNN"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2025年3月11日(星期二) 晚上10:20 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43) 
 
 
 
 
 
  是的 不用n卡其实也没啥问题 只要升级pytorch版本 英特尔的显卡也能训练，不过这个问题我已经好几年前问过了，现在受到回复还是蛮有趣的😂😂  
   
   
   
   
   
   
   
   
   
   
   
   
  东耳国王  
  ***@***.***  
   
   
   
   
   
   
  ---原始邮件---  
  发件人: ***@***.***&amp;amp;gt;  
  发送时间: 2025年3月11日(周二) 下午5:18  
  收件人: ***@***.***&amp;amp;gt;;  
  抄送: ***@***.******@***.***&amp;amp;gt;;  
  主题: Re: [PengfeiM/Fatigue-Driven-Detection-Based-on-CNN] 这个没有独显可以运行吗 (Discussion #43)  
   
   
   
   
     
  不用N卡在云服务器上能运行吧
